src/PlayingCards.tsx

```diff
@@ -3,6 +3,7 @@ import { useDrag } from "react-dnd";
 import ReactMarkdown from "react-markdown";
```

Comment:
This file and respective CSS file need to be re-named to PlayingCard (without 's', singular)